Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj

```diff
@@ -16,7 +16,8 @@
 
   <ItemGroup>
     <PackageReference Include="Ensure.That" Version="8.1.1" />
-    <PackageReference Include="Microsoft.Azure.NotificationHubs" Version="3.3.0" />
+    <PackageReference Include="Microsoft.Azure.NotificationHubs" Version="4.1.0" />
+    <PackageReference Include="Polly" Version="7.2.1" />
```

Comment:
If we upgrade the `Microsoft.Azure.NotificationHubs`, can't we use the built-in resiliency measures? I'm looking at the source code and see that it does have a built-in retry policy: https://github.com/Azure/azure-notificationhubs-dotnet/blob/main/src/Microsoft.Azure.NotificationHubs/NotificationHubClient.cs#L2836

So maybe we need to expose a knob from the wrapper library if it's not enabled by default.

Reply:
Built-in retry mechanism utilized instead Polly